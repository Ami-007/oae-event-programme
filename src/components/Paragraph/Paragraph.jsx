import React from "react";
import "./ParagraphComponent.scss";
const ParagraphComponent = (props) => {
  const { text } = props;

  return <p className="paragraph-component">{text}</p>;
};
export default ParagraphComponent;
