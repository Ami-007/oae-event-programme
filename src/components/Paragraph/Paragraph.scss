@use "../../assets/sass/variables.scss" as *;
@use "../../assets/sass/typography.scss" as *;

.paragraph {
  font-family: $modern-content;
  font-size: 1.125rem;
  padding: 0 41px;

  line-height: 27px;
}
