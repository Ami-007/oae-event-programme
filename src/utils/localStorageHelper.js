export const getDarkTheme = () => {
  //   if (darkTheme == true) {
  //     localStorage.setItem("--background-color", darkTheme);
  //   } else {
  //     localStorage.setItem("theme", "light");
  //   }
  console.log(localStorage.getItem("darkMode"));

  // convert isDarkMode to a Boolean
  // return that boolean

  return true;
};

export const setDarkTheme = (isDarkMode) => {
  localStorage.setItem("darkMode", isDarkMode.toString());
};
