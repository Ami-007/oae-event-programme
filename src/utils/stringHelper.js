export const getFontType = (fontType) => {
  if (typeof fontType === "string") {
    fontType = parseInt(fontType);
  }
  if (fontType === 1) {
    return "modern";
  } else if (fontType === 2) {
    return "classic";
  } else {
    return "modern";
  }
};

export default getFontType;
