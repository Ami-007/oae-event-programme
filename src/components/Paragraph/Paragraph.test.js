import { screen } from "@testing-library/react";
import { customRender } from "../../utils/testUtils";
import Paragraph from "./Paragraph.jsx";

it("should render the article", () => {
  customRender(<Paragraph />);
  const paragraph = screen.getByTestId("paragraph");

  expect(paragraph).toMatchSnapshot();
});
