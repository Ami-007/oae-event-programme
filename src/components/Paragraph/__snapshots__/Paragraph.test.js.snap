// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the article 1`] = `
<p
  class="paragraph"
  data-testid="paragraph"
/>
`;
